spec/rmagick/draw/class_methods/_dummy_img__spec.rb

```diff
@@ -0,0 +1,27 @@
+class Magick::Draw
+  def self._dummy_img_
+    @@_dummy_img_
+  end
+end
+
+RSpec.describe Magick::Draw, '._dummy_img_' do
+  let(:draw) { described_class.new }
+
+  it 'works' do
+    # initially this variable is not defined.
```

Comment:
Maybe we could now remove the commented out code?

Reply:
Sure, cleaned up.